### PR TITLE
Fix rejectVersionsIf vs. rejectVersionIf

### PR DESCRIPTION
**Pull request by Vampire**

### CODE REVIEW

**Review by jmfayard — APPROVED** [review]
That's great, thanks a lot

File: plugins/dependencies/src/main/resources/removed-dependencies-versions-keys.txt [hunk 1]
```diff
@@ -107,3 +107,4 @@ androidx.health..health-connect-client=version.androidx.health-connect-client
 com.google.accompanist..accompanist-glide=version.google.accompanist
 com.google.accompanist..accompanist-imageloading-core=version.google.accompanist
 app.cash.molecule..molecule-testing=version.app.cash.molecule
+androidx.compose.runtime..runtime-tracing=version.androidx.compose.runtime
```
Comment (jmfayard) [inline plugins/dependencies/src/main/resources/removed-dependencies-versions-keys.txt:110]: what's the reason for this commit?

Comment (Vampire) [reply plugins/dependencies/src/main/resources/removed-dependencies-versions-keys.txt:110]: https://github.com/Splitties/refreshVersions/actions/runs/5467565767 vs. https://github.com/Splitties/refreshVersions/actions/runs/5469518362

Comment (jmfayard) [reply plugins/dependencies/src/main/resources/removed-dependencies-versions-keys.txt:110]: I see thanks!

**Comment by jmfayard** [comment]
Thanks a lot Björn @Vampire

**Pull request merged**